core/src/main/java/ch/cyberduck/core/transfer/download/AbstractDownloadFilter.java

```diff
@@ -284,7 +284,7 @@ public void complete(final Path file, final Local local,
                     if(log.isInfoEnabled()) {
                         log.info(String.format("Append segment %s to %s", segmentFile, local));
                     }
-                    segmentFile.copy(local, new Local.CopyOptions().append(true));
+                    segmentFile.channelCopy(local, new Local.CopyOptions().append(true));
```

Comment:
Revert to `copy` when previous implementation is removed and new implementation takes previous signature.

Reply:
fixed